tests/nightly/nightly-kickoff.sh

```diff
@@ -33,135 +53,30 @@ tar xvzf $input.tar.gz -C bin/ --strip 1
-
-mv *.log 5-4-High-Availability
-mv *.zip 5-4-High-Availability
-Comment
-
-drone exec --trusted -e test="pybot -d 5-5-Heterogenous-ESXi tests/manual-test-cases/Group5-Functional-Tests/5-5-Heterogenous-ESXi.robot" -E nightly_test_secrets.yml --yaml .drone.nightly.yml
-if [ $? -eq 0 ]
-then
-HeterogenousStatus="Passed"
-else
-HeterogenousStatus="FAILED!"
-fi
-
-mv *.log 5-5-Heterogenous-ESXi
-mv *.zip 5-5-Heterogenous-ESXi
-
-drone exec --trusted -e test="pybot -d 5-6-1-VSAN-Simple tests/manual-test-cases/Group5-Functional-Tests/5-6-1-VSAN-Simple.robot" -E nightly_test_secrets.yml --yaml .drone.nightly.yml
-if [ $? -eq 0 ]
-then
-VSANStatus="Passed"
-else
-VSANStatus="FAILED!"
-fi
-
-mv *.log 5-6-1-VSAN-Simple
-mv *.zip 5-6-1-VSAN-Simple
-
-drone exec --trusted -e test="pybot -d 5-6-2-VSAN-Complex tests/manual-test-cases/Group5-Functional-Tests/5-6-2-VSAN-Complex.robot" -E nightly_test_secrets.yml --yaml .drone.nightly.yml
-if [ $? -eq 0 ]
-then
-VSANComplexStatus="Passed"
-else
-VSANComplexStatus="FAILED!"
-fi
-
-mv *.log 5-6-2-VSAN-Complex
-mv *.zip 5-6-2-VSAN-Complex
-
-drone exec --trusted -e test="pybot -d 5-7-NSX tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.robot" -E nightly_test_secrets.yml --yaml .drone.nightly.yml
-if [ $? -eq 0 ]
-then
-NSXStatus="Passed"
-else
-NSXStatus="FAILED!"
-fi
-
-mv *.log 5-7-NSX
-mv *.zip 5-7-NSX
-
-drone exec --trusted -e test="pybot -d 5-8-DRS tests/manual-test-cases/Group5-Functional-Tests/5-8-DRS.robot" -E nightly_test_secrets.yml --yaml .drone.nightly.yml
-if [ $? -eq 0 ]
-then
-DRSStatus="Passed"
-else
-DRSStatus="FAILED!"
-fi
-
-mv *.log 5-8-DRS
-mv *.zip 5-8-DRS
-
-drone exec --trusted -e test="pybot -d 5-10-Multiple-Datacenter tests/manual-test-cases/Group5-Functional-Tests/5-10-Multiple-Datacenter.robot" -E nightly_test_secrets.yml --yaml .drone.nightly.yml
-if [ $? -eq 0 ]
-then
-MultipleDCStatus="Passed"
-else
-MultipleDCStatus="FAILED!"
-fi
-
-mv *.log 5-10-Multiple-Datacenter
-mv *.zip 5-10-Multiple-Datacenter
-
-drone exec --trusted -e test="pybot -d 5-11-MultipleCluster tests/manual-test-cases/Group5-Functional-Tests/5-11-Multiple-Cluster.robot" -E nightly_test_secrets.yml --yaml .drone.nightly.yml
-if [ $? -eq 0 ]
-then
-MultipleClusterStatus="Passed"
-else
-MultipleClusterStatus="FAILED!"
-fi
-
-mv *.log 5-11-MultipleCluster
-mv *.zip 5-11-MultipleCluster
-
-if [[ $DistributedSwitchStatus = "Passed" && $ClusterStatus = "Passed" && $EnhancedLinkedModeStatus = "Passed" && $HeterogenousStatus = "Passed" && $VSANStatus = "Passed" && $VSANComplexStatus = "Passed" && $NSXStatus = "Passed" &&  $DRSStatus = "Passed" && $MultipleDCStatus =  "Passed" && $MultipleClusterStatus = "Passed" ]]
+nightlystatus=()
+count=0
+
+for i in $nightly_list_var; do
+    echo "Executing nightly test $i"
+    $test_var="pybot -d $i tests/manual-test-cases/Group5-Functional-Tests/$i.robot"
+    drone exec --trusted -e test=$test_var -E nightly_test_secrets.yml --yaml .drone.nightly.yml
+
+    if [ $? -eq 0 ]
```

Comment:
I think you need to check if nightlystatus is false as well otherwise it will only contain the last result always.